internal/chart/post_render.go

```diff
@@ -0,0 +1,78 @@
+package chart
+
+import (
+	"bytes"
+	"context"
+	"strings"
+
+	"helm.sh/helm/v3/pkg/postrender"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/kustomize/api/krusty"
+	kTypes "sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+var _ postrender.PostRenderer = &postRender{}
```

Comment:
Is this anonymous var needed? What's it do?

Reply:
Davis reminded my of implicit interface implementation via Slack.

Reply:
This is used to "enforce" that postRender implements the interface postrender.PostRenderer. It will give a more verbose error when compiling if the interface is not implemented. https://stackoverflow.com/questions/33089523/how-to-mark-golang-struct-as-implementing-interface